src/gui/DatabaseTabWidget.cpp

```diff
@@ -63,6 +63,10 @@ DatabaseTabWidget::DatabaseTabWidget(QWidget* parent)
     connect(autoType(), SIGNAL(autotypePerformed()), SLOT(relockPendingDatabase()));
     connect(autoType(), SIGNAL(autotypeRejected()), SLOT(relockPendingDatabase()));
     // clang-format on
+
+#ifdef Q_OS_MACOS
+    macUtils()->setDatabaseTabWidget(this);
```

Comment:
I do not like a util function holding onto a long-lived pointer. I would rather the MacUtils class have a signal that the DatabaseTabWidget can connect to the lock database slot. When the notification comes in, it emits the signal.

Reply:
That's what I first tried but the linker has troubles finding the signal. I'm giving it a second try and see if I can solve it.

Reply:
Got it. The `AppKit.h` header must be included in the makefile.